doc/specs/stdlib_linalg.md

```diff
@@ -206,3 +206,271 @@ program demo_outer_product
     !A = reshape([3., 6., 9., 4., 8., 12.], [3,2])
 end program demo_outer_product
 ```
+
+## `is_square` - Checks if a matrix is square
+
+### Status
+
+Experimental
+
+### Description
+
+Checks if a matrix is square
+
+### Syntax
+
+`d = [[stdlib_linalg(module):is_square(interface)]](A)`
+
+### Arguments
+
+`A`: Shall be a rank-2 array
```

Comment:
instead of A, I would suggest to use the term `array`, which is more explicit to me.
```suggestion
`d = [[stdlib_linalg(module):is_square(interface)]](array)`

### Arguments

`array`: Shall be a rank-2 array
```

Reply:
I'm not a huge fan of this change since I think the documentation is already sufficiently (and exceedingly) clear, and having a variable (even an example variable) named after a built in structure seems like a bad idea.
I could very well be the odd one out on this though, so if other people like the proposed change I'm happy to make it.

Reply:
Ok. Keep it like it is now.